lib/_http_agent.js

```diff
@@ -5,6 +5,8 @@ const util = require('util');
 const EventEmitter = require('events');
 const debug = util.debuglog('http');
 
+const emitError = require('internal/util').emitError;
```

Comment:
Personally I find that particular ES6 syntax confusing because it always looks like an object literal to me.